generate_report.py

```diff
@@ -0,0 +1,31 @@
+from helpers import login
+from helpers.helper import (
+    fetch_all_active_courses,
+    fetch_all_course_blocks,
+    fetch_block_data,
+    generate_report
+)
+
+# Login the user and fetch the session cookies
+cookie_jar = login.login_to_lms()
```

Comment:
```suggestion
session = login.login_to_lms()
```

Nit: it seems to be more self-explanatory.